package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "remimi",
-  "version": "1.0.0",
+  "version": "1.3.0",
```

Comment:
You can also drop this commit, as the next version will likely be `1.1.0` but this will be done when we `npm publish`